internal/kafka/internal/services/quota/internal/utils/simple_billing_model_resolver.go

```diff
@@ -0,0 +1,141 @@
+package utils
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared/utils/arrays"
+	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+/********************************************************************************************************************
+ * This resolver is used when the user didn't specify the `billing model` and the `marketplace` in the kafka request.
+ * It implements the legacy logic: if the user has quota for `standard`, it returns `standard`. If he has quota for
```

Comment:
`It implements the legacy logic`

Will this legacy logic go away? 
1. If not, then it's not really a legacy logic and it'll be good to update the comment to avoid future confusions
2. If yes, then let's keep track of this in an JIRA issue as well to not forget about this

Reply:
We will probably simplify the logic in the future. A Jira already exists for that. I will however remove the 'legacy logic' text.